atlasdb-timelock-server/src/integTest/resources/paxosSingleServer.yml

```diff
@@ -0,0 +1,12 @@
+algorithm:
+  type: paxos
+  paxosDataDir: var/test/data
```

Comment:
(Has been replaced with a placeholder because I switched to using `@TemporaryFolder`. But isn't this a relative path itself?)